### PR TITLE
[WIP] App Store Homepage Redesign

### PR DESCRIPTION
**Pull request by cwackerfuss**
This is a first iteration of a homepage redesign to get users looking at Blockstack apps right away.
Please let me know how this looks via the netlify link below, and if I should make any changes!

Thanks guys!

### CODE REVIEW

**Comment by shea256** [comment]
Awesome work @cwackerfuss! Will jump in and provide some notes on copy.

**Comment by shea256** [comment]
Also why is the netlify deploy check failing?

**Comment by cwackerfuss** [comment]
Taking a look at the netlify deploy now

giphy

**Comment by cwackerfuss** [comment]
2018-02-20-15-51-localhost_3000
Full Page Screenshot ^

File: app/js/pages/WhatIsBlockstack.js [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+          <section className="features">
+            <div className="container">
+              <div className="feature">
+                <div className="feature-number">1</div>
+                <div className="feature-body">
+                  <div className="row">
+                    <div className="col-md-6">
+                      <h2 className="font-weight-bold">Own your data</h2>
+                      <p>
+                        Right now, app users are "digital serfs" and applications are the "digital landlords". Users don't own their data; the app owns it. Users don't control where data gets stored; they’re only allowed to store it on the application.
```
Comment (larrysalibra) [inline app/js/pages/WhatIsBlockstack.js]: we should use "smart" punctuation:

"digital serfs" -> “digital serfs”
don't -> don’t

Comment (larrysalibra) [inline app/js/pages/WhatIsBlockstack.js]: period inside quotes & change to smart quotes

the "digital landlords". -> the “digital landlords.“

File: app/styles/main.css [hunk 2]
```diff
@@ -889,13 +889,12 @@ footer a:hover, footer a:focus, footer a:active {
 .btn {
```
Comment (larrysalibra) [inline app/styles/main.css]: on touch interfaces, touching the button results in a weird state with black text and a weird outline:
img_3260


on an iphone-size screen, the text and content being left justified looks weird - especially the app buttons - can we try centering those?

**Comment by larrysalibra** [comment]
@cwackerfuss @yknl Another thing - the button style changes on the homepage changed the button styles on the other pages - not sure if that was intentional if someone checked to make sure they all look acceptable with the new button styles.

**Comment by yknl** [comment]
Last commit addressed the alignment and copy issues pointed out by @larrysalibra

**Comment by cwackerfuss** [comment]
Quick update on this: the designs are essentially final at this point after addressing feedback, just waiting for Ryan to get back so we can get some finalized copy written and approved for both pages. I don't want us to lose momentum so any help pushing this through to the finish line is more than welcome! :slightly_smiling_face:

Invision docs:
https://invis.io/WVFVH0LH8ZQ

**Comment by cwackerfuss** [comment]
@shea256 @yknl @larrysalibra ready for eyes on copy and any general feedback.

**Comment by cwackerfuss** [comment]
@larrysalibra @yknl Copy is tentatively approved by @shea256 and this is ready for a code review.

@muneeb-ali and @shea256, will need your final approval before deploying. Thanks!

**Comment by yknl** [comment]
@cwackerfuss it's looking great!

I just have a few points on the copy and design which might be subjective. Let me know your thoughts. cc @aulneau @larrysalibra @shea256 @jeffdomke 

image
- We should keep the Logo + Wordmark (Blockstack) in the mobile view nav bar. The emphasis here is the bolded "Own your data", which lacks context. Makes me think, is this site called "Own your data"?
- It's also not very clear here to the average person how I don't I already own my data. Maybe we should add something along the lines of don't let "big web" own your data.
- I think centering the content in the hero section works better for mobile. We're doing that in the "What is Blockstack" page and it works well. Also, consistency.

___
image

- It looks like the side panel, with its limited amount of content, could just be a single line that goes in or under the hero section. It will give more emphasis to the apps and make the designer cleaner. 

___
image
- The way the apps are aligned to the left feels a little off here. Maybe aligning them to center here could fix that too. It doesn't look that bad on a computer, but the effect is more apparent when viewed from an actual phone screen. (See below)

image

- We should have more margin between the edge of the screen and the app icons, and less margin between the app icons.
- Another idea is to put them inside containers like in the Blockstack browser app screen.

___

image
- Make the column of text in the hero section a little wider on wide screens.
- We might want to mention identity in the text, since it's an important part of Blockstack

___
image
Maybe we can align things a bit more here
image

**Comment by cwackerfuss** [comment]
@shea256 please check out Ken's comments above about copy. I addressed his other points concerning style already... if its okay, I think we should launch _without_ the sidebar on the homepage. Right now its not helpful and I think we should think a little deeper about how to convey the messaging.

**Pull request merged**